BDArmory/Targeting/ModuleTargetingCamera.cs

```diff
@@ -1310,13 +1310,24 @@ public void GroundStabilize()
                 }
                 else
                 {
+                    KerbalEVA hitEVA = rayHit.collider.gameObject.GetComponentUpwards<KerbalEVA>();
+                    Part p = hitEVA ? hitEVA.part : rayHit.collider.GetComponentInParent<Part>();
+
+                    if (!hitEVA)
+                    {
+                        var pMissile = p.FindModuleImplementing<MissileLauncher>();
```

Comment:
We want to try and avoid using FindModuleImplementing<>(), especially in something getting called every FixedUpdate().
if you absolutely need the ``MissileLauncher``, use ``VesselModuleRegistry.GetModules<MissileBase>(vessel)`` instead

Reply:
Fair enough, I want to consider the possibility of users wanting to target missiles with the camera so I'm not altogether sure we can remove the need for getting MissileLauncher, though I suppose we can just require the use of a radar to do that.